database/migrations/2023_02_11_172857_add_international_columns_to_travel.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('travel', static function (Blueprint $table): void {
+            $table->boolean('is_international');
+            $table->boolean('export_controlled_technology');
```

Comment:
Should the booleans for things that (seem like they) are only relevant to international travel also be nullable? Seems odd to set them to a value on a domestic trip, for instance